### ISSUE TITLE
Allow users to repay a minimum amount greater than their debt

### ISSUE THREAD

**Issue by gomesalexandre** [shapeshift/web#6239]
### Overview

Blocked by https://gitlab.com/thorchain/thornode/-/issues/1863

Currently, when the debt is currently too small (e.g because of previously borked full repayments, or because the user deposited too small of an amount.

However, once THOR API will be able to return a minimum amount instead, we should be able to use this.

Q: We don't technically need a quote here as long as we send a Tx to the right memo with the inbound address for a given chain - are we able to do one better than THOR's API here and instead of waiting for this to be merged upstream, send to the inbound address with an arbitrary picked safe minimum?

### References and additional details

https://gitlab.com/thorchain/thornode/-/issues/1863

### Acceptance Criteria

- Users are able to send a safe minimum amount to be able to close out a loan, despite their debt being too small

**Comment by gomesalexandre** [comment]
@twblack88 let's do it, this was a nice-to-have, but with lending going hasta la vista, no point
